storage/engine/mvcc.go

```diff
@@ -1047,7 +1045,7 @@ func MVCCIncrement(engine Engine, ms *MVCCStats, key roachpb.Key, timestamp roac
 	defer iter.Close()
 
 	var int64Val int64
-	err := mvccPutUsingIter(engine, iter, ms, key, timestamp, nil, txn, func(value *roachpb.Value) ([]byte, error) {
+	err := mvccPutUsingIter(engine, iter, ms, key, timestamp, noValue, txn, func(value *roachpb.Value) ([]byte, error) {
```

Comment:
I've been thinking about this pattern as well. Could save all of these random inline comments - `noTxn`, ....